.github/workflows/ci_cron.yml

```diff
@@ -328,3 +328,36 @@ jobs:
       - name: Annotate errors
         if: ${{ failure() }}
         uses: ./.github/actions/annotate
+
+  gem_notifications:
+    runs-on: ubuntu-20.04
+    env:
+      gems:
+        "activerecord 
+        bunny
+        dalli
+        delayed_job
+        excon
+        http
+        httpclient
+        mongo
+        puma
+        sidekiq
+        sinatra
+        tilt
+        rack
+        rails
+        rake 
+        redis
+        resque
+        unicorn"
+    steps:
+      - uses: ruby/setup-ruby@v1.90.0
+        with:
+          ruby-version: 2.7
+      - uses: actions/checkout@v2
+      - run: gem install httparty
+      - name: Check for outdated gems 
+        run: ruby ./.github/workflows/scripts/slack_gem_notifications/notifications_script.rb ${{ env.gems }}
```

Comment:
The leading `./` is unnecessary (though it doesn't hurt anything if you deliberately included it for style).

Reply:
Oh cool! I didn't know that. Will remove.